src/numberGenerator.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = function generateRandomNumber(MAX_DIGITS, MAX_NUMBER) {
+  let numbers = [...Array(MAX_NUMBER).keys()];
```

Comment:
not so good decision, because you everything will get [0, 1, 2, 3], need to make it more random digits